src/chartPlugins.js

```diff
@@ -0,0 +1,46 @@
+export const doughnutDefaults = {
+  primaryLabel: {
+    text: '',
+    font: 'Arial',
+    color: '#5e5e5e',
+    size: 38,
+    offsetFromCenter: 10
+  },
+  secondaryLabel: {
```

Comment:
NIT: how about splitting to doughnutPrimaryLabelStyles and doughnutSecondaryLabelStyles?
Even though those two could just be two consts inside the gauge component itself to be near their sole consumer.
If you feel that the component got too large for a single file, then it might be time to have a folder for it.

Reply:
@thgreasi I decalre them in the `chartPlugins` mostly for fallback reasons. Should I move them in the `Gauge` component?